upload-win-packages/upload.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import boto3
+from botocore.exceptions import ClientError
+import os
+import urllib.request
+from github import Github
+from github import Auth
+
+
+def download_win_packages_from_gh_release(token, tag_name):
+    auth = Auth.Token(token)
+    fluent_bit_package_repo = Github(auth).get_repo('newrelic/fluent-bit-package').get_releases()
+    asset_names = []
+
+    for release in fluent_bit_package_repo:
+        if release.tag_name == tag_name:
+            for asset in release.get_assets():
+                if "windows" in asset.name:
+                    urllib.request.urlretrieve(asset.url, asset.name)
+                    asset_names.append(asset.name)
+
+    return asset_names
```

Comment:
Alternatively, you can download the packages like we do in [here](https://github.com/newrelic/fluent-bit-package/blob/main/ansible/provision-and-execute-tests/roles/install_fluent_bit_from_gh_prerelease/tasks/windows.yml#L1-L4).  You can pass either an explicit file like we do in [here](https://github.com/newrelic/fluent-bit-package/blob/main/ansible/provision-and-execute-tests/roles/install_fluent_bit_from_gh_prerelease/vars/main.yml#L1). To know which packages to download, you can create a `windowsStrategyMatrix` in [here](https://github.com/newrelic/fluent-bit-package/blob/main/versions/Makefile#L7-L8) (we used to have this in the past), and just download/upload those packages.